Mastodon.API/MastodonApi.cs

```diff
@@ -14,10 +14,10 @@ public class MastodonApi : IMastodonApi
         readonly MastodonApiConfig config;
         readonly Dictionary<string, string> authorizationHeader;
 
-        public MastodonApi(MastodonApiConfig config, HttpClient httpClient)
+        public MastodonApi(MastodonApiConfig config, HttpClient httpClient = null)
         {
             this.config = config;
-            apiBase = new ApiClientBase(config.InstanceUrl, httpClient);
+            apiBase = new ApiClientBase(config.InstanceUrl, httpClient ?? GetDefaultHttpClient());
```

Comment:
Thinking about this some, I will probably want to make use of IDisposable so the GC can do its job.